src/cdk/portal/dom-portal-outlet.ts

```diff
@@ -93,6 +102,29 @@ export class DomPortalOutlet extends BasePortalOutlet {
     return viewRef;
   }
 
+  /**
+   * Attaches a DOM portal by transferring its content into the outlet.
+   * @param portal Portal to be attached.
+   */
+  attachDomPortal(portal: DomPortal) {
+    // @breaking-change 9.0.0 Remove check and error once the
+    // `_document` constructor parameter is required.
+    if (!this._document) {
+      throw Error('Cannot attach DOM portal without _document constructor parameter');
+    }
+
+    let element = portal.element;
+    let anchorNode = this._document.createComment('dom-portal');
```

Comment:
Add a comment that explains what this comment node is being used for
(here and in the other `attachDomPortal`)